### PR TITLE
add docs

### PR DESCRIPTION
**Pull request by nikogu**

### CODE REVIEW

File: docs/guide/reactive-progressive.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
+# Use Status Management Progressive
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: State Management

File: docs/guide/reactive-progressive.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+> If you want to use whole Goldfish Status Management, please visit [Use Status Management](./reactive.md).
+
+## Alipay mini program status management
+
+The original framework of Alipay mini program provides the page/component level data status management ([this.setData()](https://docs.alipay.com/mini/framework/page-detail#pageprototypesetdatadata-object-callback-function)), but that is not available for page or component across scenarios. The general solution is to put data into globalData to solve the data sharing problem across pages.
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: 用英文文档链接。

Comment (yibuyisheng) [reply docs/guide/reactive-progressive.md]: page -> Page
component -> Component

File: docs/guide/reactive-progressive.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+It can be seen that basic mechanism is available for status management in the original Alipay mini program, which does not suffice the status management in complicated projects.
+
+## Goldfish responsive status management
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: Reactive State Management

File: docs/guide/reactive-progressive.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+On basis of the related great ideas in the community, Goldfish deposits a set of mini program friendly responsive status management framework, which enables the user to flexibly and efficiently handle the status data.
+
+In Goldfish, status data is responsive, you can use `@goldfishjs/reactive-connect` declare responsive data, and use `AppStore` `PageStore` `ComponentStore` declare store, finally use `createApp` `createPage` `createComponent` to connect store data.
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: you can use `observable`, `state`, `computed` to create reactive data.

File: docs/guide/reactive-progressive.md (hunk 2 above)
Comment (nikogu) [reply docs/guide/reactive-progressive.md]: 哪里有英文文档链接。

Comment (nikogu) [reply docs/guide/reactive-progressive.md]: done

File: docs/guide/reactive-progressive.md (hunk 4 above)
Comment (nikogu) [reply docs/guide/reactive-progressive.md]: done

File: docs/guide/reactive-progressive.md (hunk 3 above)
Comment (nikogu) [reply docs/guide/reactive-progressive.md]: done

File: docs/guide/reactive-progressive.md (hunk 1 above)
Comment (nikogu) [reply docs/guide/reactive-progressive.md]: done

File: docs/guide/getting-start.md [hunk 5]
```diff
@@ -7,4 +7,7 @@
 
 ## Use project template
 
-TODO
+* Use template from IDE.
```
Comment (yibuyisheng) [inline docs/guide/getting-start.md]: templates

File: docs/guide/getting-start.md [hunk 6]
```diff
@@ -7,4 +7,7 @@
...
+* Clone github repository: https://github.com/alipay/goldfish.git
```
Comment (yibuyisheng) [inline docs/guide/getting-start.md]: GitHub

Comment (yibuyisheng) [reply docs/guide/getting-start.md]: 句尾加上结束标点：.

File: docs/guide/reactive-progressive.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
+# Use State Management Progressive
+
+> If you want to use whole Goldfish state Management, please visit [Use state Management](./reactive.md).
+
+## Alipay mini program State Management
+
+The original framework of Alipay mini program provides the Page/Component level data state management ([this.setData()](https://docs.alipay.com/mini/framework/page-detail#pageprototypesetdatadata-object-callback-function)), but that is not available for Page or Component across scenarios. The general solution is to put data into globalData to solve the data sharing problem across pages.
+
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: globalData -> `globalData`

File: docs/guide/reactive-progressive.md [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+It can be seen that basic mechanism is available for state management in the original Alipay mini program, which does not suffice the state management in complicated projects.
+
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: which is not satisfied with the state management in complicated projects.

File: docs/guide/reactive-progressive.md [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+## Goldfish Reactive State Management
+
+On basis of the related great ideas in the community, Goldfish deposits a set of mini program friendly responsive state management framework, which enables the user to flexibly and efficiently handle the state data.
+
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: responsive -> reactive

File: docs/guide/reactive-progressive.md [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+In Goldfish, state data is responsive, you can use `observable`, `state`, `computed` to create reactive data, and use `AppStore` `PageStore` `ComponentStore` declare store, finally use `createApp` `createPage` `createComponent` to connect store data.
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: responsive -> reactive

Comment (yibuyisheng) [reply docs/guide/reactive-progressive.md]: finally -> then

Comment (yibuyisheng) [reply docs/guide/reactive-progressive.md]: to connect store data -> to connect the store with the view instance (Page, Component, or App)

File: docs/guide/reactive-progressive.md [hunk 11]
```diff
@@ -0,0 +1,205 @@
...
+```js
+// component.js
+import { createComponent } from '@goldfishjs/core';
+import store from './store';
+
+Component(createComponent(store, {
+  didMount() {},
+  methods: {
+    onPopupClose(event) {
+      // responsive data
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: reactive

File: docs/guide/reactive-progressive.md [hunk 12]
```diff
@@ -0,0 +1,205 @@
...
+      this.store.closeCount++;
+    },
+  }
+}));
+```
+
+Direct reference is possible in the component:
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: possible -> available

File: docs/guide/reactive-progressive.md [hunk 13]
```diff
@@ -0,0 +1,205 @@
...
+
+When connect store to Page, directly reference is possible in the Page and Component:
+
+```xml
+<view>{{currentShop}}</view>
+```
+
+### Change Data
+
+To change data in the responsive system is even more simple -- just modify data directly and then all places with the data used are updated synchronously:
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: responsive -> reactive

Comment (yibuyisheng) [reply docs/guide/reactive-progressive.md]: -- -> -

File: docs/guide/reactive-progressive.md [hunk 14]
```diff
@@ -0,0 +1,205 @@
...
+...
+
+globalData.orderInfo.money = 200;
+
+...
+```
+
+### Use computed & watch
+
+In more cases when your attributes need calculation or listen, the “computed” and “watch” can be used. For more concepts, see [Vue Computed Properties and Watchers](https://vuejs.org/v2/guide/computed.html#ad):
```
Comment (yibuyisheng) [inline docs/guide/reactive-progressive.md]: “computed” -> *computed*
“watch” -> *watch*

**Pull request merged**